src/options.cpp

```diff
@@ -180,9 +192,11 @@ options parse_options(int argc, char** argv) {
       } else {
         next_vim_file_bufnr = bufnr;
       }
-    } else if (parser.match_flag_option("--help"sv, "--h"sv)) {
+    } else if (parser.match_flag_option("--help"sv, "--h"sv) ||
+               parser.match_flag_shorthand("-h"sv)) {
```

Comment:
Shorthand options should always be a single character, so should we just give a single character to the function?
```suggestion
    } else if (parser.match_flag_option("--help"sv, "--h"sv) ||
               parser.match_flag_shorthand('h')) {
```